index.d.ts

```diff
@@ -22,6 +22,13 @@ export type Options = {
 	@default 'https://github.com'
 	*/
 	readonly baseUrl?: string;
+
+	/**
+	Additional reference prefix to support. It can be set to `undefined` to disable the default.
```

Comment:
Not sure the point of supporting `undefined` to disable default? What will it even do then?